### DISCUSSION TITLE
How to save voxel grid or octree

### DISCUSSION THREAD

**Discussion by XiongFenghhh** [isl-org/Open3D#5659] [Q&A] [answered]
Hello, I followed the [instruction of voxelization](http://www.open3d.org/docs/release/tutorial/geometry/voxelization.html). How can I save the voxel grid(or octree) and visualize it in other software(such as blender,meshlab,maya)

**Comment by xing-yuu** [comment, marked as answer]
o3d.io.write_voxel_grid(name,voxel_grid, write_ascii=True)
